tests/framework/cli_exec.go

```diff
@@ -0,0 +1,74 @@
+package framework
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	. "github.com/onsi/gomega"
+
+	ktests "kubevirt.io/kubevirt/tests"
+)
+
+type Result struct {
+	cmd           string
+	verb          string
+	resourceType  string
+	resourceName  string
+	resourceLabel string
+	filePath      string
+	nameSpace     string
+	query         string
+	expectOut     string
+	actualOut     string
+	params        []string
+}
+
+func execute(r Result) string {
+	return executeWithCustomTimeout(r, LongTimeout)
```

Comment:
why LongTimeout as default?

Reply:
my previous thought is user specify their own timeout value case by case, but anyway I can change it to a more proper way